Activity.js

```diff
@@ -2,21 +2,52 @@ class Activity {
   constructor(category, description, minutes, seconds) {
     this.category = category;
     this.description = description;
-    this.minutes = minutes;
-    this.seconds = seconds;
+    this.minutes = parseInt(minutes);
```

Comment:
This was a much needed addition, to ensure that the correct data type (integer) is being passed in.